### DISCUSSION TITLE
Diffusivity at extrapolated temp in mS/cm

### DISCUSSION THREAD

**Discussion by thienbinh92** [materialsproject/pymatgen#2535] [Q&A] [answered]
Hello, I found in the [user guide](https://pymatgen.org/pymatgen.analysis.diffusion_analyzer.html?highlight=diffusion_analyzer) that the function " get_extrapolated_diffusivity Returns (float) Diffusivity at extrapolated temp in mS/cm." I think diffusivity obtained from this function should be cm-2/s, not mS/cm, shouldn't it?
<img width="588" alt="Screenshot 0004-05-29 at 20 13 18" src="https://user-images.githubusercontent.com/56110825/170865134-a60f70b6-3f17-4d0b-94f7-f3d8ac42634d.png">

**Comment by shyuep** [comment, marked as answer]
Yes you are correct. I have fixed this in the doc for pymatgen-analysis-diffusion.
